### PR TITLE
Refractor

### PR DESCRIPTION
**Pull request by redwing555**
In the following milestone for the school library project @chaw-bot  and @redwing555  did the following : 

- YAGNI, KISS and DRY rules and Ruby style guide are respected.
- Following SOLID design principles.
- Paying special attention to the "Single-responsibility Principle".
- Make sure that the role of `main.rb` is only to start your code and ask the user for the initial choice of options (or to quit).
  Extract some pieces of logic into a separate class(es).

### CODE REVIEW

File: lib/main.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
+require_relative './methods/create_book'
```
Comment (rloterh) [inline lib/main.rb]: - [x] OPTIONAL
It's a good practice to keep the `main.rb` in the _root directory_ of your project, so as to make it easily accessible.

File: lib/main.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+    @options.each do |index, string|
+      puts "#{index} - #{string}"
+    end
+    print "\nEnter option from above list: "
+    gets.chomp.to_i
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
```
Comment (rloterh) [inline lib/main.rb]: - [x] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hello Team, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Recommended standards/ principles applied in code refactoring
- Application is working as intended
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @redwing555,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**